### DISCUSSION TITLE
Fire and forget bunches

### DISCUSSION THREAD

**Discussion by maxoriggi** [ttscoff/bunch-docs#17] [Ideas]
Bunches keeps track of open and closed bunches, but it's also useful to just launch scripts (or auto launch them with the new front matter stuff) to perform actions which do not leave open apps around.
In this situation the open status doesn't matter and in certain situations could also cause issues such as with the 'Remember Open Bunches' option which would relaunch the script at startup just because it was used early on.

In this context it would be nice to have a new front matter option to skip it: it could be called 'open status' (default true), if false the bunch won't be marked as open when launched and everything else should keep working fine.

**Comment by ttscoff** [comment]
I have added `toggles: false` as an option to the frontmatter for b69. You can also use `only opens: true` if that makes more semantic sense. But it covers this use case. Bunches with this key will not get a checkmark at all, won't register as open, and won't affect other Bunches when those close... because the latest version also won't close apps that are launched by another open Bunch, unless you specify `quit apps: always` for the Bunch. Docs updated here: http://brettterpstra.com/bunch-beta/docs/bunch-files/frontmatter/#behavior

**Reply by maxoriggi** [reply]
Yep and it works nicely, I like the switch back to circles for markers and the fact that they do not show up in this case, it makes it easy to distinguish the various bunches.
